src/api/image/src/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const cors = require('cors');
+const helmet = require('helmet');
+
+const service = require('./service');
+
+const app = express();
+app.use(helmet());
+app.use(cors());
+
+// TODO: figure out how to do logging for each service...
```

Comment:
This sounds like a logging service which aggregates ALL service logs to a central location. In this case, I imagine that would be either an HTTP call, or a log rotate which gradually shifts logs from the service to the central location.

Reply:
We have Kibana for this, but aren't actively using it yet.  I'd like all of our services to dump logs into it, so I guess it's just a matter of reading in ENV variables to set the Kibana details (like we do now in the root).